frontend/dockerfile/linter/linter.go

```diff
@@ -5,16 +5,81 @@ import (
 	"strings"
 
 	"github.com/moby/buildkit/frontend/dockerfile/parser"
+	"github.com/pkg/errors"
 )
 
+type LinterConfig struct {
```

Comment:
`s/LinterConfig/Config/` as prefix is redundant as it is already in the package name